src/Sentry/SentryOptions.cs

```diff
@@ -333,6 +333,12 @@ public int MaxCacheItems
         /// <see href="https://github.com/getsentry/sentry-dotnet/issues/71"/>
         public bool RequestBodyCompressionBuffered { get; set; } = true;
 
+        /// <summary>
+        /// Whether to send client reports, which contain statistics about discarded events.
+        /// </summary>
+        /// <see href="https://develop.sentry.dev/sdk/client-reports/"/>
+        public bool SendClientReports { get; set; } = true;
```

Comment:
`m`: Where did you see this option? Do other SDKs do this as well? If so, I also need to add this on Cocoa.

Reply:
I was assuming we'd want a way to turn them off.  Perhaps not?

Reply:
I don't think that we want to turn that off. At least the other SDKs also don't have this; see https://github.com/getsentry/sentry-python/pull/1181/files and https://github.com/getsentry/sentry-javascript/pull/3955/files. What do you think @bruno-garcia, @marandaneto, and @adinauer?

Reply:
Worth asking on client-infra, I'm not aware of it as well.

Reply:
@denrase please follow this thread for either adding it or not, I will reply once I have an answer.

Reply:
@philipphofmann we do have this option on most web SDKs
* https://github.com/getsentry/sentry-python/blob/17ea78177d605683695352783750f24836c4e620/sentry_sdk/consts.py#L79
* https://github.com/getsentry/sentry-ruby/blob/c14c43a560722436c4d9b7b3f6ae2bb193f3caa2/sentry-ruby/lib/sentry/configuration.rb#L205-L208
* https://github.com/getsentry/sentry-javascript/blob/f715e8754c11797120989e52e2342c89b98b5ea1/packages/types/src/options.ts#L130-L134

Reply:
Cool, who's gonna update the specs? https://develop.sentry.dev/sdk/client-reports/
Thanks!

Reply:
PR here: https://github.com/getsentry/develop/pull/544